kitchen/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM containersol/mesos-agent:1.0.0-0.1.0
 
+RUN apt-get update
```

Comment:
It started out as a separate app but we consolidated since we wanted to be able to run both the demo and kitchen via minimesos. With a single Dockerfile the minimesos agents can choose to support only one of the apps: demo or kitchen.

Also, two separate apps: we have two different features: image search and image tagging.

Reply:
Created separate docker image for the tensorflow demo app